MetaMask/metamask-extension
# feat: MCv1 Permissions: Site Connections Tooltip

## **Description**
A follow-up PR to add tooltips to ConnectionListItem.



[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://codespaces.new/MetaMask/metamask-extension/pull/23189?quickstart=1)

## **Related issues**

Fixes: https://github.com/MetaMask/MetaMask-planning/issues/1980

## **Manual testing steps**

1. Open Metamask
2. Connect to a dapp on at least 8 accounts
3. Go To "All Permissions" screen from the global menu item
4. Hover over an Account AvatarGroup 
5. Verify that the tooltip shows a maximum of 7 accounts with blockies/jazzicons and has a "+X more" representing the accounts we could not show in the tooltip.

## **Screenshots/Recordings**



### **Before**
No tooltip


### **After**
<img width="358" alt="image" src="https://github.com/MetaMask/metamask-extension/assets/10986371/158aa934-c22d-4094-890e-f6ab81bd31b1">



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
